documentation/projects/proposals/dark_mode/20240313-project_proposal_dark_mode.md

```diff
@@ -0,0 +1,81 @@
+# 2024-03-13 Project Proposal
+
+**Author**: @fcoveram
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @obulat
+- [ ] @zackkrida
+
+## Project summary
+
+<!-- A brief one or two sentence summary of the project's features -->
+
+Include a feature that changes the UI from light to a dark color scheme.
+
+### Motivation
+
+Dark mode is a customization level that is in the spirit of adapting the UI to
+people’s contexts. The benefits span a11y, personal preferences set on the
+devices, energy efficiency in devices, and an alternative browsing experience of
+visual content due to the background contrast.
+
+## Goals
+
+<!-- Which yearly goal does this project advance? -->
+
+Refine Search Experience
+
+## Requirements
```

Comment:
In addition to the design documentation requirements you listed, this section should include the requirements of the dark mode functionality itself. Some possible examples:

- The user can toggle dark,light, or system mode through an easy-to-find user interface control
- The location of that UI control (header? footer?)
- What the default value should be (light or system?)
- Goals for the dark mode itself:
  - Preserve the visual identity of Openverse, sufficient contrast, etc. 

The goal of this section of the plan is that if, hypothetically, another designer had to take on this work, they would have a good understanding of your intentions.

Reply:
I am not sure about the second point as it specifies an UI decision that can be discussed on the design ticket once is created. The first point covers the design approach needed more broadly and gives room to design iteration.

For the remaining points:
* The default setting is something I missed, thanks for surfacing it.
* I will add the color contrast as an a11y requirement.
* Preserving the brand identity is a great idea, adding it.

Thanks for the suggestions.